File: exercises/D101/solution.pas
var
    a, b: longint;
begin
    readln(a);
    b := a div 10000000;
    if (b = 2) then
        writeln('Fixed')
    else if (b = 3) then
        writeln('Fixed')
    else
        writeln('Mobile');
end.

File: exercises/D101/solution.c
#include <stdio.h>
int main() {
    int a;
    scanf("%d", &a);
    int b = a / 10000000;
    if (b == 2) {
        printf("Fixed\n");
    } else if (b == 3) {
        printf("Fixed\n");
    } else {
        printf("Mobile\n");
    }
    return 0;
}
